### ISSUE TITLE
Blog has landing page, posts are on subpage, not showing in Joomla

### ISSUE THREAD

**Issue by opal9** [cognidox/WordBridge#78]
Wordbridge is successfully merging several blogs onto my Joomla site. I love it.

I've just created a new blog, and a new menu item with Woodbridge set up properly, but since the blog posts are on a subpage of the new blog (that is, I have a landing page), the posts aren't being pulled by Wordbridge.

The posts are on blogname.wordpress.com/subpage . Is there a way to make this happen?
Thank you.

**Comment by lbourne43** [comment]
HI opal9

Could you just confirm that your blog is hosted on wordpress.com and not self-hosted? This will affect how the component pulls in posts. On a self-hosted site you should just be able create a menu entry with your blog name set to blogname.example.com/subpage.

Regards
Lewis Bourne

**Comment by opal9** [comment]
Good point, but no, it's not self-hosted. I have eleven other WP.com blogs that Wordbridge is handling beautifully, but they all have the blog posts front and center. This latest one is the only one I have with a landing page.
